src/pages/create-quiz-page/create-quiz.tsx

```diff
@@ -1,7 +1,180 @@
+          <i className="pi pi-file"></i>
+          <p>2 answers</p>
+        </div>
+      </div>
+    </div>
+  );
+
+  return (
+    <div className="create-quiz flex flex-column">
+      <Header />
+      <div className="card--center mt-6">
+        <Card
+          header={HeaderCard1}
+          footer={FooterCard1}
+          className="create-quiz__first-card create-quiz__common-card-style"
+        ></Card>
+      </div>
+
+      <div className="flex flex-row card--center mt-6 gap-3">
+        <Card
+          header={HeaderCard2}
+          footer={FooterCard2}
+          className="create-quiz__second-card create-quiz__common-card-style"
+        >
+          <div className="flex flex-column align-items-start create-quiz__answers-wrapper">
+            <div className="create-quiz__answer">
+              <div className="create-quiz__input-label">Question</div>
+              <InputText
+                style={{ width: "640px" }}
+                className="create-quiz__input"
+                placeholder="What is your question?"
+              />
+            </div>
+
+            <div className="create-quiz__answer">
+              <div className="create-quiz__input-label">Answer 01</div>
+              <div className="flex flex-row align-items-center gap-3">
+                <InputText
+                  style={{ width: "312px" }}
+                  className="create-quiz__input"
+                  placeholder="Enter your answer"
+                />
+                <i className="pi pi-trash"></i>
+              </div>
+              <div className="field-radiobutton mt-2">
+                <RadioButton
+                  inputId="answer1"
+                  name="answer"
+                  value="Answer1"
+                  onChange={(e) => setAnswer(e.value)}
+                  checked={answer === "Answer1"}
+                />
+                <label htmlFor="answer1">This is the correct answer</label>
+              </div>
+            </div>
+
+            <div className="create-quiz__answer">
+              <div className="create-quiz__input-label">Answer 02</div>
+              <div className="flex flex-row align-items-center gap-3">
+                <InputText
+                  style={{ width: "312px" }}
+                  className="create-quiz__input"
+                  placeholder="Enter your answer"
+                />
+                <i className="pi pi-trash"></i>
+              </div>
+              <div className="field-radiobutton mt-2">
+                <RadioButton
+                  inputId="answer2"
+                  name="answer"
+                  value="Answer2"
+                  onChange={(e) => setAnswer(e.value)}
+                  checked={answer === "Answer2"}
+                />
+                <label htmlFor="answer2">This is the correct answer</label>
+              </div>
+            </div>
+
+            <div className="create-quiz__answer">
+              <Button
```

Comment:
The button components used in the app have some repetitive styles. For example here the same class as button-cancel is used. I suggest creating a general button component which has different styles applied based on the prop types (error, general, outlined, etc)